pastastore/connectors.py

```diff
@@ -1223,7 +1223,7 @@ def _write_pstore_config_file(self):
         config = {
             "connector_type": self.conn_type,
             "name": self.name,
-            "path": self.parentdir,
+            "path": os.path.abspath(self.parentdir),
```

Comment:
Path(self.parentdir).resolve()? :)

Reply:
original solution also fine of course

Reply:
currently no pathlib is used in pastastore, so sticking to os.path for now, 
But i'm open to moving to pathlib, but we should open a new issue for that